new/LIT7040HassabaHegg.xml

```diff
@@ -0,0 +1,66 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7040HassabaHegg" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title  xmlns:t="http://www.tei-c.org/ns/1.0" xml:lang="gez" xml:id="t1">Ḥassāba ḥǝgg</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="#BLorient9798"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Perpetual calendar of the movable feasts based on the great lunar cycle of 532 years in columns or tables. The columns contain: 
+                    (1) <foreign xml:lang="gez">አው፡ ቀመ፡ </foreign>, (2) <foreign xml:lang="gez">ዓው፡ ዐመ፡ </foreign>, 
+                    (3) <foreign xml:lang="gez">ዕጥ፡ ለን፡</foreign>, (4) <foreign xml:lang="gez">አበቅቴ፡ </foreign>, 
+                    (5) <foreign xml:lang="gez">መጥቅዕ፡ </foreign>, (6) <foreign xml:lang="gez">ነነዌ፡ </foreign>, 
+                    (7) <foreign xml:lang="gez">በዓተ፡ ጾም፡</foreign>, (8) <foreign xml:lang="gez">ፍሥሕ፡</foreign>, 
+                    (9) <foreign xml:lang="gez">ፋሲካ፡</foreign>. In the left margin historical notes (<foreign xml:lang="gez">ታሪክ፡</foreign>) 
+                    are added. They seem to be written by at least two different hands at different times. The last note concerns the death of
```

Comment:
Should information about hands be copied into a LIT file? I would say it belongs to the manuscript description.

Reply:
Yes, but the information is sensitive, because it leads us to a judgement, when it was composed. I will probably rephrase it.